css-canvas-medical/styles.css

```diff
@@ -0,0 +1,225 @@
+body {
+  width: 960px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 810px;
+}
```

Comment:
Here is where you can add your bottom border. Lemme know if it doesn't work somehow.

https://developer.mozilla.org/en-US/docs/Web/CSS/border-bottom